Bottom: 6e7e62f469359edd83b1d880a39c91bfdf89fed3
Top:    021491e87d7a4a576c6d2f7f97025d6ef1155a41
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 14:25:45 +0000

w32: Update cairo recipe to 1.10.2

(Probably not the latest available, but one I successfully
used with gEDA and PCB before)


---

diff --git a/w32/recipes/cairo.recipe b/w32/recipes/cairo.recipe
index 8654d5c..62cf803 100644
--- a/w32/recipes/cairo.recipe
+++ b/w32/recipes/cairo.recipe
@@ -1,8 +1,11 @@
 name=cairo
-version=1.8.8
+version=1.10.2
 release=1
 download=http://www.cairographics.org/releases
 
 case "$host" in
-  *mingw*) configure_opt=--without-x ;;
+  *mingw*)
+    configure_opt=--without-x
+    CPPFLAGS="-D ffs=__builtin_ffs"
+  ;;
 esac
